cmd/build.go

```diff
@@ -275,6 +275,15 @@ func getBuildOpts(cmd *cobra.Command) ([]leeway.BuildOption, *leeway.FilesystemC
 		_ = os.MkdirAll(coverageOutputPath, 0644)
 	}
 
+	var dockerBuildOptions *leeway.DockerBuildOptions
+	dockerBuildOptionsArg, _ := cmd.Flags().GetString("docker-build-options")
```

Comment:
this flag is not registered in the `init` function, hence will never be available